CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contribution Guidelines
+
+## Reporting issues
+
+- **Search for existing issues.** Please check to see if someone else has reported the same issue.
+- **Share as much information as possible.** Include operating system and version, browser and version. Also, include steps to reproduce the bug. [This](http://github.com/mozilla/webmaker-app/ISSUE_BOILERPLATE.md) is a good boilerplate to use.
+
+## Project Setup
+
+Refer to the [documentation](http://webmaker-mobile-guide.mofodev.net/_docs/get_started/README.html).
+
+## Code Style
+
```

Comment:
Could mention the existence of an `.editorconfig` file at the root level which is supported by most editors and IDEs.